.chloggen/transform-name-suffix.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: processor/transform
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Introduce optional metric name suffix setup for convert summary and extract histogram functions"
```

Comment:
Could we just say "metric conversion functions" here and in the subtext list the affected functions and the new default suffix?